workflow/executor/executor.go

```diff
@@ -421,19 +421,42 @@ func (we *WorkflowExecutor) SaveLogs() (*wfv1.Artifact, error) {
 	return &art, nil
 }
 
+// Retrive the Secrets from VolumeMount
```

Comment:
Function comments should use name of function in sentence. e.g.:
```
// GetSecretFromVolMount retrieves the secrets from the volume mount
```
https://blog.golang.org/godoc-documenting-go-code